### PR TITLE
mockup for applicaiton summary page completed

### PR DESCRIPTION
**Pull request by SoLetsDev**
## Title
ECER-1683: Brief description of the changes

## Description

-router addition and also new page for application summary
-included small bug fix for education validation.

## Checklist
- [x] I have tested these changes locally.
- [ ] Changes to backend endpoints are covered by passing integration tests.
- [ ] I have added or updated the necessary documentation.

## Screenshots (if applicable)
/application/summary
image

wizard 
image

### CODE REVIEW

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ApplicationSummary.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        },
+      ],
+    };
+  },
+  computed: {},
+  methods: {},
+});
+</script>
+<style scoped>
+.test {
```
Comment (pcatkins) [inline src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ApplicationSummary.vue:53]: Do we need this?

**Review by pcatkins — CHANGES_REQUESTED** [review]
Small clean up.

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ApplicationSummary.vue (hunk 1 above)
Comment (SoLetsDev) [reply src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ApplicationSummary.vue:53]: .test .... nope haha damn. nice catch

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/router.ts [hunk 2]
```diff
@@ -69,6 +69,11 @@ const router = createRouter({
       component: () => import("./components/pages/LogoutCallback.vue"),
       meta: { requiresAuth: false },
     },
+    {
+      path: "/application/summary",
```
Comment (pcatkins) [inline src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/router.ts:73]: suggesting we go with `/manage-application`, the way it is now feels like a child route of application. Thoughts?

**Review by pcatkins — APPROVED** [review]

**Pull request merged**